setup.cfg

```diff
@@ -1,3 +1,130 @@
+[metadata]
+name = opsdroid
+license = Apache License 2.0
+url = https://opsdroid.github.io/
+download_url = https://github.com/opsdroid/opsdroid/releases
+author = Jacob Tomlinson
+author_email = jacob@tom.linson.uk
+description = "An open source ChatOps bot framework."
+long_description = file: README.md
+long_description_content_type = text/markdown
+platforms = any
+classifiers =
+    Development Status :: 4 - Beta
+    Environment :: Console
+    Framework :: AsyncIO
+    Intended Audience :: Developers
+    Intended Audience :: System Administrators
+    Intended Audience :: Information Technology
+    License :: OSI Approved :: Apache Software License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Topic :: Communications :: Chat
+    Topic :: Scientific/Engineering :: Artificial Intelligence
+    Topic :: Software Development :: Libraries :: Python Modules
+keywords =
+    bot
+    bot-framework
+    opsdroid
+    botkit
+    python3
+    asyncio
+    chatops
+    devops
+    nlu
+
+[options]
+include_package_data = True
+zip_safe = False
+packages = find:
+test_suite = tests
+setup_requires = babel
+install_requires =
+	aiohttp>=3.6.2
+	appdirs>=1.4.4
+	arrow>=0.15.8
+	Babel>=2.8.0
+	click>=7.1.2
+	ibm-watson>=4.4.1
+	multidict>=4.7.6
+	nbconvert>=5.6.1
+	nbformat>=5.0.6
+	opsdroid-get-image-size>=0.2.2
+	parse>=1.16.0
+	puremagic>=1.9
+	pycron>=1.0.0
+	pyyaml>=5.3.1
+	regex>=2020.7.14
+	tailer>=0.4.1
+	websockets>=8.1
+	voluptuous>=0.11.7
+	watchgod>=0.6;python_version>="3.6"
+
+[options.packages.find]
+exclude =
+	tests
+	tests.*
+	modules
+	modules.*
+	docs
+	docs.*
+
+[options.entry_points]
+console_scripts =
+	opsdroid = opsdroid.cli:cli
+
+[options.extras_require]
+# connectors
+connector_matrix =
+	bleach>=3.1.5
+	matrix-nio>=0.14.1
+connector_mattermost =
+	mattermostdriver>=7.0.1
+connector_slack =
+	certifi>=2020.4.5.2
+	slackclient>=2.7.3
+	emoji>=0.6.0
+connector_webex =
+	webexteamssdk>=1.6
+# parsers
+parser_dialogflow =
+	dialogflow>=0.8.0
+# TODO: Add this module back in when it can handle ImportError as the dialogflow module does
+#parser_watson =
+#	ibm-watson>=4.4.1
+# databases
+database_redis =
+	aioredis>=1.3.1
+database_sqlite =
+	aiosqlite>=0.15.0
+database_mongo =
+	motor>=2.1.0
+# testing
+test =
+	flake8>=3.8.1
+	black>=19.10b0
+	coveralls>=2.0.0
+	dialogflow>=0.8.0
+	astroid>=2.4.1
+	pytest>=5.4.2
+	pytest-aiohttp==0.3.0
+	pytest-asyncio>=0.12.0
+	pytest-cov>=2.7.1
+	pytest-mock>=3.2.0
+	pytest-timeout>=1.4.0
+	pydocstyle>=5.0.2
+	asynctest>=0.13.0
+	mypy-lang>=0.5.0
+	sphinx>=3.0.4
+	sphinx-click>=2.3.2
+	recommonmark>=0.6.0
+	deadlinks>=0.3.2
```

Comment:
it would probably be good to move these into a "docs" requirement, and use that over "test" for read the docs. Installing less things on RTD means a faster build.

Reply:
so the config should just use these 4?

Reply:
no test should be all the others, and these should be docs.

Reply:
yea i meant the RTD CI check

Reply:
seems like pytest also needs to be in docs?

Reply:
looks like yeah